### PR TITLE
Powershell content support

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Description
This PR adds support for Powershell script type on content side.
It includes common server scripts (which still require implementation - TBD in the following weeks), package support, schema support and common server docs generation.

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Documentation/common_server_docs.py [hunk 1]
```diff
@@ -156,18 +157,74 @@ def createPyDocumentation(path, origin, language):
...
+            'origin': origin
+        }
+        function_name = function[0]
+        function_doc['name'] = function_name
+        parameters = function[1].split('.PARAMETER')
+
+        description = parameters[0].split('.DESCRIPTION')[1].strip()
+        if not description:
+            isErrorPS = True
+            print("Missing description for in PS function {}.\n".format(function_name))
```
Comment (yaakovi) [inline Documentation/common_server_docs.py]:
```suggestion
            print("Missing description for PS function {}.\n".format(function_name))
```

File: Tests/scripts/pkg_dev_test_tasks.py [hunk 2]
```diff
@@ -209,6 +209,8 @@ def main():
         script_obj = script_obj.get('script')
     script_type = script_obj.get('type')
     if script_type != 'python':
+        if script_type == 'powershell':
```
Comment (yaakovi) [inline Tests/scripts/pkg_dev_test_tasks.py]: why is that different from javascript?

**Comment by yardensade** [comment]
Please note that powershell file extension does not have to be .ps1, we need to be careful about that

**Comment by ronykoz** [comment]
@yardensade which file extensions should we support?

**Comment by Itay4** [comment]
We will support only ps1 for now

File: Tests/scripts/hook_validations/structure.py [hunk 3]
```diff
@@ -43,6 +43,7 @@ class StructureValidator(object):
         REPORT_REGEX,
         SCRIPT_PY_REGEX,
         SCRIPT_JS_REGEX,
+        SCRIPT_PS_REGEX,
```
Comment (ronykoz) [inline Tests/scripts/hook_validations/structure.py]: what about INTEGRATION_PS_REGEX

**Pull request merged**